Doc: How we schedule meetings (#10)
hyphacoop/handbook: protocols.md

# Protocols

> > Protocols are the "attitudes" or "the manner in which one approaches each and

Reviewer: See my comment on "protocol" as a term (feeling a little technologizing / scientizing), that said I'm totally down with gug of the idea of protocols

Author: Totally agree. What about stealing "guides" from enspiral?

Author: used "guides" though we're being a little more specific than enspiral treats guides

every element in our space." They are different than pure rules or
instructions; they are ways that we establish order and maintain practices
across our group [...] -- They define the way we ought to proceed or behave in
different situations. As such, they are normative, or premised on values,
morals, and an idea of how things ought to be done. They are a manifestation of
our values.
>
> -- [Lab Book, Civic Laboratory for Environmental Action Research (CLEAR)][clear-lab-book]

## Scheduling a meeting

Note: Consider whether this meeting might be a global interest to other
  members. Skip below steps at your discretion for low-stakes topics.

- Try to announce your intention to schedule a meeting on a topic, and ask

>   who's interested to participate. Allow a day for people to respond.

Reviewer: ```suggestion
  who's interested to participate. Allow _at least_ a day for people to respond.
```

I'd say if our internal turnaround time is 48hours 1 day is a short minimum

Author: ```suggestion
  who's interested to participate. Allow _at least_ 48 hours for people to respond.
```

Your commit suggestion didn't align with my interpretation of your concern. Should I accept mine or yours? :)

Author: (Also, in general, should we air toward hours or days with this stuff -- specificity or intentional generality?)

Reviewer: go for yours. don't think days/hours matters with this given our specificity elsewhere

- Consider who might be most interested, and what timing might work for
  them.
  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
  - Check everyone's [weekly availability `schedules`][schedules].

>     for them.

Reviewer: ```suggestion
```

- If 2-3 candidate timeslots feel obvious from looking at internal
  availabilities:
  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
    Infrastructure Meeting (Option 1)`
  - Ask in chat whether anyone who wants to attend has any blocks on any
    of the options.
- If short-notice or speed of booking is important:
  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
    - If time permits or topic isn't an all-hands matter, consider
      leaving the hold available for someone else.
    - If claiming a hold, edit the calendar immediately, and announce
      meeting in chat.
- If your must find a new timeslot:

>   - Create a [new When2Meet poll](https://www.when2meet.com/).

Reviewer: Maybe say "Create a new poll" (we currently use [When2Meet](#))

Author: Good call. Committed locally.

>   - Try to leave ample time for completion (up to 1 week?).

Reviewer: ```suggestion
  - Leave ample time for completion (ideally 1 week).
```

Author: Committed

>   - Check on non-responders in following days, and poke them.

Reviewer: ```suggestion
  - Check on non-responders in following days, and send (gentle) reminders as needed.
```

## Hosting a meeting

- **Before** the meeting...
  - create a stub agenda as soon as possible (can be done before scheduling)

>     - Hint: use [an agenda `template`][template].

Reviewer: ```suggestion
    - Hint: use [an agenda `template`][template]
```

Author: done, and related question below :)

>   - add the agenda to the [`meetings` index][meetings].

Reviewer: ```suggestion
  - add the agenda to the [`meetings` index][meetings]
```

>   - update the [organizational calendar][calendar] as event scheduling

Reviewer: Unclear what this specifically means, maybe rephrase? 

"as soon as the event is scheduled"

Author: Ah, these used to be closer in an earlier version, but the "Scheduling a meeting" section advises holding a few slots (like you did for design jam). The idea was that the calendar should always reflect options and finalized times.

In light of that, maybe "update the organizational calendar as early as possible with candidate and finalized time slots"?

Reviewer: :+1:

Author: ```suggestion
  - update the [organizational calendar][calendar] as early as possible with candidate and finalized time slots
```

>     progresses.

Reviewer: ```suggestion
    progresses
```
